### PR TITLE
Create an attachement for CBA report on states Bids Selection and Bids S...

### PR DESCRIPTION
**Pull request by yvaucher**
...elected

### CODE REVIEW

**Comment by gurneyalex** [comment]
:+1:

**Comment by lepistone** [comment]
Thanks :+1: . It's ok like this, but maybe you could do a domain instead of computed fields, like this:

https://github.com/odoo/odoo/blob/8.0/addons/account_voucher/account_voucher.py#L311-314

**Comment by yvaucher** [comment]
@lepistone Good suggestion, I'll change it with domains

File: purchase_requisition_bid_selection/report.xml [hunk 1]
```diff
@@ -7,6 +7,7 @@
       id="action_report_purchase_requisitions"
       model="purchase.requisition"
       report_type="qweb-pdf"
+      attachment="(object.state in ('open', 'selected') and len(object.eligible_bid_ids)) and ('CBA-' + (object.name or '') + '-' + time.strftime('%Y%m%d%h%M') + '.pdf')"
```
Comment (yvaucher) [inline purchase_requisition_bid_selection/report.xml]: Hours should be %H not %h

**Comment by yvaucher** [comment]
@lepistone I Improved the way fields for reports where computed getting ride of those functions.

**Pull request merged**